epouta/epouta-pilot-guide.rst

```diff
@@ -88,10 +88,6 @@ mostly follow the `cPouta User Guide
 <https://research.csc.fi/pouta-user-guide>`_ for instructions on how to use
 OpenStack.
 
-When you login, you need to specify a user domain. This is always "users":
```

Comment:
This is also mentioned further down under "Differences compared to cPouta" so it should be removed from there as well.